super_team_picker/routes/posts.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const knex = require('../db/client.js');
+
+const router = express.Router()
+
+//------------------POST ROUTES---------------------------
+
+//------------------------Index of all Posts: ---------------
+
+// the below path automatically assumes that is has the '/posts' prefixed to it
```

Comment:
again, in real practice people who will be reading your code are experienced enough to know what you are doing. I suggest start practicing on being able to tell what to comment on.